quiz.js

```diff
@@ -0,0 +1,86 @@
+function check() {

```

Comment:
In my final layout, I created a javascript file for the quiz to further work on the functionality aspect of my website. The quiz is fully functioning but is not 100% complete as there as still placeholder questions.